css/app.css

```diff
@@ -215,6 +215,18 @@ html, body {
   padding-left: 3px;
 }
 
+#graphiql-container .tool-button {
+  cursor: pointer;
+  text-decoration: none;
+  font-family: sans-serif;
+  color: #ccc;
+  padding: 2px 4px;
+  margin: 0 4px;
+}
+#graphiql-container .tool-button:hover {
+  color: #333;
```

Comment:
I intentionally make the button lighter when it's not hover so it doesn't pollute too much the UI . I'm happy to implement any other design